### PR TITLE
Add TVTime

### PR DESCRIPTION
**Pull request by Reprevise**
I added the data for TVTime in the entertainment YML file and the icon of TVTime in the entertainment images folder.

### CODE REVIEW

**Review by theoricher — APPROVED** [review]
I approve this change

**Review by theoricher — DISMISSED** [review]
It's ok for me

File: _data/entertainment.yml [hunk 1]
```diff
@@ -178,6 +178,14 @@ websites:
       facebook: TiVo
       img: tivo.png
       tfa: No
+      
+    - name: TVTime
+      url: https://wwwtvtime.com
```
Comment (kmpoppe) [inline _data/entertainment.yml]:
```suggestion
      url: https://www.tvtime.com
```
Typo in URL

Comment (theoricher) [reply _data/entertainment.yml]: > 
> 
> Typo in URL

Yes ! :)

**Review by Carlgo11 — COMMENTED** [review]
How does this fit the site criteria?

**Comment by kmpoppe** [comment]
@EcoGames what site criteria (https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#site-criteria) does TVTime.com match?

**Comment by Reprevise** [comment]
Users can create profiles on TVTime which allows users to upload profile photos, sometimes of themselves.

File: _data/entertainment.yml [hunk 2]
```diff
@@ -173,6 +173,13 @@ websites:
     facebook: TiVo
     img: tivo.png
 
+  - name: TVTime
```
Comment (Carlgo11) [inline _data/entertainment.yml]:
```suggestion
  - name: TV Time
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**